### ISSUE TITLE
DEV-25: add React Styleguidist to the project

### ISSUE THREAD

**Issue by KovDimaY** [KovDimaY/MERN-ChatBot#39]
https://github.com/styleguidist/react-styleguidist

Components to add:

- [ ] ~Initial page (Welcome page)~
- [ ] ~Chatbot~
- [ ] ~Header~
- [x] **Message**
- [x] **Quick Reply**
- [x] **Cards**
- [x] **Common**
  - [x] CardItem
  - [x] QuickReplyItem
  - [x] CertificationItem
  - [x] DotsAnimation
  - [x] EducationItem
  - [x] EmptyView
  - [x] ExperienceItem
  - [x] HobbyItem
  - [x] LanguageItem
  - [x] ProjectItem

To work on this issue, please use a branch **DEV-25** created from **devel**. When the work is finished - create a PR to the **devel** branch. At the end of each development circles, code from the **devel** branch (if everything is ok) goes to the **master** branch and deploys to production.

**IMPORTANT!** Before creating a PR be sure that all the tests are passing and the coverage is 100%

**Comment by KovDimaY** [comment]
https://github.com/dialogs/dialog-web-components
https://dialogs.github.io/dialog-web-components

I  can use them as a great example of how to build a styleguidist in my project

**Comment by KovDimaY** [comment]
Closed accidentally

**Comment by KovDimaY** [comment]
In the end, due to technical problems related to the MaterialCSS library connected to my project, I decided not to add the ChatBot component. Regarding the **Header** and **Initial Page** components I decided that thay are not needed to add as they are not components as they are so it does not make a lot of sense to have them documented. So, with this, I think I can deploy the code and make the release to have it accessible on PROD.
